### ISSUE TITLE
Error: (unix time) try if you are using GNU date

### ISSUE THREAD

**Issue by wangyuyy** [nerfstudio-project/nerfstudio#1933]
**Describe the bug**
I tried to use nerfstudio to generate customized dataset. The error occurred while running colmap command.

**To Reproduce**
1. Install the nerfstudio as tutorial.
2. Running command
```shell
ns-process-data images --data rgb --output-dir processed
```
3. QT error
```shell
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
qt.qpa.screen: QXcbConnection: Could not connect to display localhost:0
Could not connect to any X display.
```
4. solved QT error
```shell
export QT_QPA_PLATFORM=offscreen
```
5. Unix time error

**Expected behavior**
Expected to output processed data.

**Screenshots**
<img width="1105" alt="image" src="https://github.com/nerfstudio-project/nerfstudio/assets/106719785/acdc1434-1b62-44ec-9c6b-bdaf1231a9cc">

**Comment by tancik** [comment]
I don't think this issue is related to the date, that is just info if you want to see the date in a different format. Have you been able to run colmap on one of the datasets that is known to work (ie the poster dataset). Can you also try running with `--no-gpu`

**Comment by hrz2000** [comment]
The same error can be run with --no-gpu, but this is too slow. How can you use gpu without reporting an error?
